### DISCUSSION TITLE
Deploy Nextjs on Shared Hosting

### DISCUSSION THREAD

**Discussion by Khaan25** [vercel/next.js#64215] [Help]
### Summary

Hey Guys, As you know Nextjs requires Nodejs...

I've a Shared Windows Hosting and enabled Nodejs there but still I'm having issues deploying there...

- First Issue is regarding the case sensitive of file system while generating a build on the Shared Hosting.
- Any guides to deploy Nextjs on Shared Hosting on Plesk/cPanel?

### Steps to Reproduce

![WhatsApp Image 2024-04-07 at 11 08 21_94d02081](https://github.com/vercel/community/assets/72739794/cd846fd6-7692-4d9e-b0bd-b3af03e5c218)

I've shared an Image of the issue when I try to build the nextjs app on the Windows Shared Hosting Plesk

**Comment by amyegan** [comment]
Hey @Khaan25. This looks like it may be an issue with your hosting provider rather than a Next.js issue. You may need to work with them directly to find a solution. I found a few resources that might help.
- [How to install and run Node.js application in Plesk?](https://www.plesk.com/kb/support/how-to-install-and-run-node-js-application-in-plesk/)
- [How to Host Node.js Applications on Plesk Obsidian](https://www.plesk.com/blog/product-technology/node-js-plesk-onyx/)
- [support.plesk.com](https://support.plesk.com)
